website/src/models/challenge.js

```diff
@@ -97,7 +97,6 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
   let [challengeTasks, userTasks] = await Q.all([
     // Find original challenge tasks
     Tasks.Task.find({
-      userId: {$exists: false},
```

Comment:
why this was removed? we want to select the original challenge tasks, this way it will also get the ones that are already synced to the users